numpy/core/_add_newdocs.py

```diff
@@ -1537,6 +1537,10 @@ def luf(lamdaexpr, *args, **kwargs):
 
     Set numerical operators for array objects.
 
+    .. deprecated:: 1.16
+
+        Use :c:func:`PyUFunc_ReplaceLoopBySignature` to change ufunc behaviour.
```

Comment:
Perhaps we should also suggest creating a subclass with an overloaded `__array_ufunc__` here, for python-only users who were replacing operators.

Reply:
done

Reply:
This reads badly. Maybe
```
        For the general case, use :c:func:`PyUFunc_ReplaceLoopBySignature`.
        For ndarray subclasses, define the ``__array_ufunc__`` method and
        override the relevant ufunc.
```

Reply:
Ping.